pkgs/applications/video/mapmap/default.nix

```diff
@@ -37,6 +40,14 @@ mkDerivation rec {
     gst_all_1.gst-vaapi
   ];
 
+  patches = [
+    (fetchpatch {
+      name = "message-handler-segfault.patch";
+      url = "https://github.com/mapmapteam/mapmap/pull/519.patch";
```

Comment:
Just one last request: Could you use `https://github.com/mapmapteam/mapmap/pull/519/commits/22eeee59ba7de6de7b73ecec3b0ea93bdc7f04e8.patch` here? This way it's obvious which commit is meant to be fetched and we won't have the problem of the PR changing later.

Reply:
Good catch. Done.